react/finger-exercises/redux/01 - actions and reducers/books/reducer.js

```diff
@@ -26,6 +27,11 @@ function reducer(state = initialState, action) {
         ...state,
         favoriteBook: action.payload.favoriteBook
       };
+    case actions.SET_MOST_HATED_BOOK:
+      return {
```

Comment:
Mmm for do this you should use `return state.merge({`. This way you merge the new param in the old state